tests/backup/backup_restore_basic_test.go

```diff
@@ -2800,3 +2801,208 @@ var _ = Describe("{BackupRestoreOnDifferentK8sVersions}", func() {
+			log.Infof("Cluster [%s] uid: [%s]", destinationClusterName, dstClusterUid)
+		})
+		Step("Create a schedule policy", func() {
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			schedulePolicyintervalInMins = 15
+			log.InfoD("Creating a schedule policy with interval [%v] mins", schedulePolicyintervalInMins)
+			schedulePolicyName = fmt.Sprintf("interval-%v-%v", schedulePolicyintervalInMins, time.Now().Unix())
+			schedulePolicyInfo := Inst().Backup.CreateIntervalSchedulePolicy(5, int64(schedulePolicyintervalInMins), 5)
+			err = Inst().Backup.BackupSchedulePolicy(schedulePolicyName, uuid.New(), orgID, schedulePolicyInfo)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of schedule policy [%s] with interval [%v] mins", schedulePolicyName, schedulePolicyintervalInMins))
+			schedulePolicyUid, err = Inst().Backup.GetSchedulePolicyUid(orgID, ctx, schedulePolicyName)
+			log.FailOnError(err, "Fetching uid of schedule policy [%s]", schedulePolicyName)
+			log.Infof("Schedule policy [%s] uid: [%s]", schedulePolicyName, schedulePolicyUid)
+		})
+		Step("Schedule applications in source cluster", func() {
+			log.InfoD("Scheduling applications in source cluster")
+			for i := 0; i < numDeployments; i++ {
+				taskName := fmt.Sprintf("src-%s-%d", taskNamePrefix, i)
+				nameSpace := fmt.Sprintf("test-namespace-%s", taskName)
+				appContexts := ScheduleApplicationsOnNamespace(nameSpace, taskName)
+				appNamespaces = append(appNamespaces, nameSpace)
+				contexts = append(contexts, appContexts...)
+				for index, ctx := range appContexts {
+					appName := Inst().AppList[index]
+					ctx.ReadinessTimeout = appReadinessTimeout
+					log.InfoD("Scheduled application [%s] in source cluster in namespace [%s]", appName, nameSpace)
+				}
+			}
+		})
+		Step("Validate app namespaces in source cluster", func() {
+			log.InfoD("Validating app namespaces in source cluster")
+			ValidateApplications(contexts)
+		})
+		Step("Create schedule backup", func() {
+			log.InfoD("Creating a schedule backup")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			scheduleName = fmt.Sprintf("%s-schedule-%v", BackupNamePrefix, time.Now().Unix())
+			labelSelectors := make(map[string]string)
+			err = CreateScheduleBackup(scheduleName, SourceClusterName, backupLocationName, backupLocationUID, appNamespaces,
+				labelSelectors, orgID, "", "", "", "", schedulePolicyName, schedulePolicyUid, ctx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of schedule backup with schedule name [%s]", scheduleName))
+			firstScheduleBackupName, err := GetFirstScheduleBackupName(ctx, scheduleName, orgID)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the name of the first schedule backup [%s]", firstScheduleBackupName))
+			scheduleNames = append(scheduleNames, scheduleName)
```

Comment:
You may consider avoiding storing the scheduleName separately in a slice since there is only one scheduleName.

Reply:
Done